platform/lib/samples/Templates.js

```diff
@@ -0,0 +1,78 @@
+/**
```

Comment:
It looks like this file is not used at all.

Reply:
removed